# Atbalsts audio log validation

One microphone session, 9 September 2026. User confirms phone-speaker playback, MacBook reception and internet disabled. This establishes the acoustic offline loop on that pair, not FM radio or phone-receiver reliability.

## Definitions

Goodput = unique accepted body bytes × 8 / full receive seconds. The burst denominator is detected candidates, not all transmitted packets. The registry was preloaded. Three-minute capacity is a linear projection, not a measurement.

In [1]:
import sys, json
from pathlib import Path
root=Path('/Users/kristaps/Documents/New project/output/atbalsts-radio-review')
sys.path.insert(0,str(root))
from analyze_log import analyze
result=analyze()
print(json.dumps(result,indent=2))

{
  "statistics": {
    "session": "3ikpsj6w",
    "exported_utc": "2026-09-09T14:11:04.070Z",
    "active_seconds": 30.68,
    "first_object_seconds": 19.05779999999702,
    "detected_bursts": 24,
    "frames_ok": 17,
    "frames_failed": 7,
    "detected_frame_success_fraction": 0.7083333333333334,
    "objects_verified": 1,
    "status_records_applied": 180,
    "body_bytes": 904,
    "signed_object_bytes": 968,
    "unique_fragments_needed_for_object": 9,
    "body_bits_per_second": 235.72359843546283,
    "body_bytes_per_second": 29.465449804432854,
    "registry_preloaded": 803,
    "facilities_with_status": 180,
    "registry_status_coverage_fraction": 0.22415940224159403,
    "situations_received": 0,
    "peak_max": 3.1908,
    "windows_above_one": 5,
    "high_peak_time_range": [
      6,
      7
    ],
    "linear_180_second_projection_bytes_NOT_measured": 5303.780964797914
  },
  "outcomes": [
    {
      "outcome": "Decoded",
      "count": 17,
      "detected_total": 24,


## Interpretation

180 status changes arrived after 19.06 seconds; the complete-session rate was 235.72 bits/s. Seven of 24 detected bursts failed, but one complete signature-checked object was applied. The log cannot establish the intended transmission size, RF performance or reliability across phones. The level spike does not establish the cause of failures.